### PR TITLE
Create mainfest.json

### PR DESCRIPTION
**Pull request by Pikaswelt**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [ ] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [ ] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [ ] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/crazymine/mainfest.json
Comment (maaaathis) [inline minecraft_servers/crazymine/mainfest.json:1]: Rename this file to `manifest.json`

File: minecraft_servers/crazymine/mainfest.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
+{
+  "server_name": "crazymine",
+  "nice_name": "CrazyMine",
+  "direct_ip": "crazymine.de",
+  "server_wildcards": [
+    "%.laby.net"
+  ],
```
Comment (maaaathis) [inline minecraft_servers/crazymine/mainfest.json:7]:
```suggestion
```

File: minecraft_servers/crazymine/mainfest.json [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  "supported_languages": [
+     "de"
+  ],
+  "social": {
+    "web": "https://web.crazymine.de",
+    "web_shop": "-",
```
Comment (maaaathis) [inline minecraft_servers/crazymine/mainfest.json:13]:
```suggestion
```

File: minecraft_servers/crazymine/mainfest.json [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    "web_support": "https://web.crazymin.de/support",
```
Comment (maaaathis) [inline minecraft_servers/crazymine/mainfest.json:14]: Not reachable

File: minecraft_servers/crazymine/mainfest.json [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+    "twitter": "-",
```
Comment (maaaathis) [inline minecraft_servers/crazymine/mainfest.json:15]:
```suggestion
```

File: minecraft_servers/crazymine/mainfest.json [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+    "tiktok": "-",
```
Comment (maaaathis) [inline minecraft_servers/crazymine/mainfest.json:16]:
```suggestion
```

File: minecraft_servers/crazymine/mainfest.json [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+    "facebook": "-",
```
Comment (maaaathis) [inline minecraft_servers/crazymine/mainfest.json:17]:
```suggestion
```

File: minecraft_servers/crazymine/mainfest.json [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+    "instagram": "-",
```
Comment (maaaathis) [inline minecraft_servers/crazymine/mainfest.json:18]:
```suggestion
```

File: minecraft_servers/crazymine/mainfest.json [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+    "discord": "https://discord.com/invite/HPdCPjyrE6",
+    "youtube": "https://www.youtube.com/@CrazyMineMC",
+    "teamspeak": "-"
```
Comment (maaaathis) [inline minecraft_servers/crazymine/mainfest.json:21]:
```suggestion
```

File: minecraft_servers/crazymine/mainfest.json [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+  "social": {
+    "web": "https://web.crazymine.de",
+    "web_shop": "-",
+    "web_support": "https://web.crazymin.de/support",
+    "twitter": "-",
+    "tiktok": "-",
+    "facebook": "-",
+    "instagram": "-",
+    "discord": "https://discord.com/invite/HPdCPjyrE6",
+    "youtube": "https://www.youtube.com/@CrazyMineMC",
```
Comment (maaaathis) [inline minecraft_servers/crazymine/mainfest.json:20]:
```suggestion
    "youtube": "https://www.youtube.com/@CrazyMineMC"
```

File: minecraft_servers/crazymine/mainfest.json [hunk 10]
```diff
@@ -0,0 +1,61 @@
...
+    "teamspeak": "-"
+  },
+  "gamemodes": {
+    "Lobby": {
+      "name": "Citybuild",
```
Comment (maaaathis) [inline minecraft_servers/crazymine/mainfest.json:25]: Key doesn't match to name

File: minecraft_servers/crazymine/mainfest.json [hunk 11]
```diff
@@ -0,0 +1,61 @@
...
+      "name": "Citybuild", 
+      "color": "#FFA500",
+      "url": "https://web.crazymine.de/citybuild",
+      "command": "/citybuild",
+      "versions": "1.20.4"
+    },
+    "dna": {
+      "name": "DNA",
+      "color": "#ADD8E6",
+      "url": "https://web.crazymine.de/boxpvp",
```
Comment (maaaathis) [inline minecraft_servers/crazymine/mainfest.json:34]: Not found

File: minecraft_servers/crazymine/mainfest.json [hunk 12]
```diff
@@ -0,0 +1,61 @@
...
+    "instagram": "-",
+    "discord": "https://discord.com/invite/HPdCPjyrE6",
+    "youtube": "https://www.youtube.com/@CrazyMineMC",
+    "teamspeak": "-"
+  },
+  "gamemodes": {
+    "Lobby": {
+      "name": "Citybuild", 
+      "color": "#FFA500",
+      "url": "https://web.crazymine.de/citybuild",
```
Comment (maaaathis) [inline minecraft_servers/crazymine/mainfest.json:27]: Not found

File: minecraft_servers/crazymine/mainfest.json [hunk 13]
```diff
@@ -0,0 +1,61 @@
...
+     "background": "#0A56A5",
+     "text": "#FFFFFF"
+  }, 
+  "location": {
+     "city": "Walldorf",
+     "country": "Germany",
+     "country_code": "DE"
+  },
+  "yt_trailer": "-",
+  "user_stats": "https://laby.net/@{userName}",
```
Comment (maaaathis) [inline minecraft_servers/crazymine/mainfest.json:59]:
```suggestion
```

File: minecraft_servers/crazymine/mainfest.json [hunk 14]
```diff
@@ -0,0 +1,61 @@
...
+     "primary": "#008FE8",
+     "background": "#0A56A5",
+     "text": "#FFFFFF"
+  }, 
+  "location": {
+     "city": "Walldorf",
+     "country": "Germany",
+     "country_code": "DE"
+  },
+  "yt_trailer": "-",
```
Comment (maaaathis) [inline minecraft_servers/crazymine/mainfest.json:58]:
```suggestion
```

**Review by maaaathis — CHANGES_REQUESTED** [review]
You need at a least a `icon.png` and a `icon@2x.png`. See https://github.com/LabyMod/server-media/blob/master/docs/Files.md

**Comment by DoJapHD** [comment]
Inactive

**Pull request closed without merging**